src/components/WriteEditModal/WriteEditModal.module.scss

```
@@ -114,3 +114,64 @@
     outline: none;
   }
 }
+
+.feedback-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 1.8rem;
```

flex-direction은 없어도 될 것 같아요. 아니면 이런 식으로...?
```suggestion
  @include flex-column(1.8rem);
  
  flex-direction: row;
```